<ui version="4.0" >
 <class>Subclassing</class>
 <widget class="QDialog" name="Subclassing" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qt Designer Form Subclassing</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Class Na&amp;me:</string>
     </property>
     <property name="buddy" >
      <cstring>comboClassName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="comboClassName" />
     </item>
     <item>
      <widget class="QPushButton" name="newImplementation" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>4</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;New...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;File Name:</string>
     </property>
     <property name="buddy" >
      <cstring>filename</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="filename" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&amp;Location:</string>
     </property>
     <property name="buddy" >
      <cstring>location</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="location" >
     <property name="readOnly" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="3" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Filter &amp;by:</string>
     </property>
     <property name="buddy" >
      <cstring>filterEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="filterEdit" />
     </item>
     <item>
      <widget class="QToolButton" name="clearButton" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../resources/resources.qrc" >:/divers/images/clear_left.png</iconset>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QTreeView" name="filterView" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="animated" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Create</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>C&amp;ancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filterEdit</tabstop>
  <tabstop>filterView</tabstop>
  <tabstop>comboClassName</tabstop>
  <tabstop>filename</tabstop>
  <tabstop>location</tabstop>
  <tabstop>newImplementation</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Subclassing</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>480</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel" >
     <x>61</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Subclassing</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>566</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel" >
     <x>331</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
